src/lib/gql/index.ts

```diff
@@ -55,23 +60,43 @@ export function getGql(url: string, getToken: () => string) {
       }
     );
 
+  const createCircleIntegration = async (
+    circleId: number,
+    type: string,
+    name: string,
+    data: any
+  ) =>
+    makeQuery(url, getToken)('mutation')(
+      {
+        insert_circle_integrations_one: [
+          {
+            object: {
```

Comment:
if you rebase on hasura-next, this has been changed from `object` to `payload`

Reply:
> Asked for help in #hasura-dev about regenerating these types

@0xchomper thinking this through, i think it's lighter-weight for all involved if we (coordinape) just update this in `hasura-next` after merge. Thanks for trying though